students/jimin/users/views.py

```diff
@@ -1,3 +1,45 @@
-from django.shortcuts import render
+import json ,re
 
+from django.http     import JsonResponse
+from django.views    import View
+
+
+from users.models    import User
 # Create your views here.
+
+class SignUpView(View):
+    def post(self, request):
+        try :
+             data          = json.loads(request.body)
+             email         = data['email']
+             password      = data['password']
+             phone_number  = data['phone_number']
```

Comment:
가장 긴 변수 명에서 1칸만  띄어 주시면 됩니다.
```suggestion
             data         = json.loads(request.body)
             email        = data['email']
             password     = data['password']
             phone_number = data['phone_number']
```